src/Import/src/FMI3/fmi3_import_capi.c

```diff
@@ -29,6 +29,18 @@ extern "C" {
 
 static const char * module = "FMILIB";
 
+
+static void fmi3_import_capi_destroy_dllfmu_and_restore_options(fmi3_import_t* fmu) {
+	if (fmu->capi->options) {
+		/* Take back ownership of Options */
```

Comment:
I think the indentation in this function needs some adjustment.

Reply:
Fixed